samples/subsys/testsuite/pytest/shell/pytest/test_shell.py

```diff
@@ -2,44 +2,33 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import time
 import logging
 
-from twister_harness import Device
+import pytest
+from twister_harness import DeviceAdapter
+from twister_harness.helpers.shell import Shell
```

Comment:
`Shell` class can be imported in `twister_harness.__init__` similar to `DeviceAdapter` to have simpler user interface.

Reply:
Done